livecheck/main.py

```diff
@@ -408,6 +408,10 @@ def do_main(*, auto_update: bool, keep_old: bool, cat: str, ebuild_version: str,
                     sp.run(('mv', ebuild, new_filename), check=True)
             with open(new_filename, 'w') as f:
                 f.write(content)
+            if hook and sp.run((hook.name, 'pre', search_dir, cp, str_old_version, str_new_version,
+                                old_sha, new_sha, hash_date),
+                               check=False).returncode != 0:
+                logger.error(f'Error running hook {hook.name}')
```

Comment:
Shouldn't this exit here?

```suggestion
                click.echo(f'Error running hook {hook.name}.', err=True)
                raise click.Abort
```